official/vision/beta/modeling/backbones/mobilenet.py

```diff
@@ -0,0 +1,682 @@
+# Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Contains definitions of EfficientNet Networks."""
+
+import math
+from typing import Text, Optional, Dict
+
+# Import libraries
+from absl import logging
+import tensorflow as tf
+from official.modeling import tf_utils
+from official.vision.beta.modeling.layers import nn_blocks
+from official.vision.beta.modeling.layers import nn_layers
+
+layers = tf.keras.layers
+regularizers = tf.keras.regularizers
+
+
+class Conv2DBNBlock(tf.keras.layers.Layer):
```

Comment:
Sure, I have moved it to nn_blocks. 

- The reason I created it is because it will help to simply the definition of architecture.
- I put it here because no other backbone use something similarly.